requirements.txt

```diff
@@ -1,5 +1,5 @@
 ####### Requirements without Version Specifiers #######
-numpy>=1.22.1
```

Comment:
I had added this while initially testing, sorry for pushing directly into your branch